pkg/operator/ceph/cluster/rbd/controller.go

```diff
@@ -214,7 +214,7 @@ func (r *ReconcileCephRBDMirror) reconcile(request reconcile.Request) (reconcile
 	// If the version of the Ceph monitor differs from the CephCluster CR image version we assume
 	// the cluster is being upgraded. So the controller will just wait for the upgrade to finish and
 	// then versions should match. Obviously using the cmd reporter job adds up to the deployment time
-	if !reflect.DeepEqual(*runningCephVersion, *desiredCephVersion) {
+	if !cephCluster.Spec.External.Enable && !reflect.DeepEqual(*runningCephVersion, *desiredCephVersion) {
```

Comment:
nit: The comment section above this block should be updated the reflect the changes to the conditional statement

Reply:
Added `// If the external cluster, Rook not need to attempt to track that (see #10696)`.